lib/matplotlib/tests/test_arrow_patches.py

```diff
@@ -119,16 +119,32 @@ def test_arrow_styles():
     styles = mpatches.ArrowStyle.get_styles()
 
     n = len(styles)
-    fig, ax = plt.subplots(figsize=(6, 10))
+    fig, ax = plt.subplots(figsize=(8, 8))
     ax.set_xlim(0, 1)
     ax.set_ylim(-1, n)
+    fig.subplots_adjust(left=0, right=1, bottom=0, top=1)
 
     for i, stylename in enumerate(sorted(styles)):
-        patch = mpatches.FancyArrowPatch((0.1, i), (0.8, i),
+        patch = mpatches.FancyArrowPatch((0.1, i), (0.45, i),
                                          arrowstyle=stylename,
                                          mutation_scale=25)
         ax.add_patch(patch)
 
+    for i, stylename in enumerate([']-[', ']-', '-[', '|-|']):
+        style = stylename
+        if stylename[0] != '-':
+            style += ',angleA=-45'
```

Comment:
possibly use angles that are *not* 45 (e.g. 30) so that the origin (vertical or horizontal) is clearer?

Reply:
OK, how's that?